oneview/resource_server_profile_template.go

```diff
@@ -2209,28 +2208,38 @@ func resourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{})
 			}
 			// Gets Connection Body
 			connections = append(connections, map[string]interface{}{
-				"allocated_mbps":         connection.AllocatedMbps,
-				"allocated_vfs":          connection.Connectionv200.AllocatedVFs,
-				"boot":                   connectionBoot,
-				"function_type":          connection.FunctionType,
-				"id":                     connection.ID,
-				"interconnect_uri":       connection.InterconnectURI.String(),
-				"ipv4":                   connectionIpv4,
-				"isolated_trunk":         connection.IsolatedTrunk,
-				"lag_name":               connection.LagName,
-				"mac_type":               connection.MacType,
-				"managed":                connection.Managed,
-				"maximum_mbps":           connection.MaximumMbps,
-				"name":                   connection.Name,
-				"network_name":           connection.NetworkName,
-				"network_uri":            connection.NetworkURI,
-				"port_id":                connection.PortID,
-				"private_vlan_port_type": connection.PrivateVlanPortType,
-				"requested_mbps":         connection.RequestedMbps,
-				"state":                  connection.State,
-				"status":                 connection.Status,
+				"boot":           connectionBoot,
+				"function_type":  connection.FunctionType,
+				"id":             connection.ID,
+				"ipv4":           connectionIpv4,
+				"isolated_trunk": connection.IsolatedTrunk,
+				"lag_name":       connection.LagName,
+				"mac_type":       connection.MacType,
+				"managed":        connection.Managed,
+				"name":           connection.Name,
+				"network_name":   connection.NetworkName,
+				"network_uri":    connection.NetworkURI,
+				"port_id":        connection.PortID,
+				"requested_mbps": connection.RequestedMbps,
 			})
 		}
+
+		conSetVal := d.Get("connection_settings").([]interface{})
+		for _, rawConSet := range conSetVal {
+			conSet := rawConSet.(map[string]interface{})
+			consVal := conSet["connections"].(*schema.Set).List()
+			for i, rawConVal := range consVal {
+				con := rawConVal.(map[string]interface{})
+				if len(connections) == len(consVal) {
+					if connections[i]["id"] == con["id"] {
```

Comment:
Add comment what you're doing here

Reply:
will need to take care of this in next PR